src/cargo/core/compiler/unit_dependencies.rs

```diff
@@ -204,6 +204,41 @@ fn deps_of<'a, 'cfg>(
     Ok(())
 }
 
+/// Gets all of the features enabled for a package, plus its dependencies'
+/// features.
+///
+/// Dependencies are added as `dep_name/feat_name` because `required-features`
+/// wants to support that syntax.
+pub fn resolve_all_features(
```

Comment:
I'm not sure why this function is moved, but I think it should stay where it originally was for now.

Reply:
Oh, I thought that `resolve_all_features` is more related to lower details in core compiler than higher ops. Since you have concerns, I'll change it back.

Reply:
@ehuss, `ops::cargo_compile` is a private mod. what should I do now?

Reply:
Should I change it to public?

Reply:
You'll need to make `resolve_all_features` pub, and re-export it around [here](https://github.com/rust-lang/cargo/blob/4d1c5edb45f9749ed2b12cb02372acfcbed0eb6a/src/cargo/ops/mod.rs#L2).